js/ui/panelMenu.js

```diff
@@ -206,6 +210,8 @@ var Button = new Lang.Class({
     },
 
     _onDestroy() {
+        this.parent();
```

Comment:
Part of this patch needs to be squashed with an earlier one (I had a comment about that)